invenio_rdm_records/records/jsonschemas/records/definitions-v1.2.0.json

```diff
@@ -35,7 +35,14 @@
     "type": "string"
   },
   "descriptionType": {
```

Comment:
`Type` vs `_type`... I see for example `titleType` got changes to `title_type` when it became a vocabulary.